manifests/m/Microsoft/Teams/1.4.00.7174/Microsoft.Teams.installer.yaml

```diff
@@ -17,8 +17,8 @@ Installers:
     Silent: /s
     SilentWithProgress: /s
 - Architecture: x86
-  InstallerUrl: https://statics.teams.cdn.office.net/production-windows/1.4.00.7174/TeamsSetup.exe
-  InstallerSha256: F6F93C930D0179BF8074D48896FF5EE8CABE12799425E93DF7F5F33163EF2220
+  InstallerUrl: https://statics.teams.cdn.office.net/production-windows/1.4.00.7174/Teams_windows.exe
```

Comment:
The `TeamsSetup.exe` installer first finds the architecture of your system and then downloads the `.nupkg` file (and extracts it) from the servers depending on your system's architecture. This means that running the `TeamsSetup.exe` installer on an `arm64` system will install arm64 version of Teams and viceversa.

This replaces the URL with the correct x86 installer.